exchanges/ftx/ftx.go

```diff
@@ -267,51 +267,75 @@ func (f *FTX) SendHTTPRequest(ep exchange.URL, path string, result interface{})
 }
 
 // GetMarginBorrowRates gets borrowing rates for margin trading
-func (f *FTX) GetMarginBorrowRates() (MarginFundingData, error) {
-	var resp MarginFundingData
-	return resp, f.SendAuthHTTPRequest(exchange.RestSpot, http.MethodGet, marginBorrowRates, nil, &resp)
+func (f *FTX) GetMarginBorrowRates() ([]MarginFundingData, error) {
+	type result struct {
```

Comment:
```go
result :=  struct {
     Data []MarginFundingData `json:"result"`
}{}
```
We can short hand these, don't get fancy and do `result :=  &struct` because then it will panic if it errors.